### PR TITLE
EVM-C: add EVM_STATIC_MODE_ERROR status code

### LINKED ISSUE #154: EVM-C: add status code for "invalid in static mode"

### PR DESCRIPTION
**Pull request by axic**
Fixes #154.

### CODE REVIEW

File: include/evm.h [hunk 1]
```diff
@@ -142,6 +142,7 @@ enum evm_status_code {
     EVM_STACK_OVERFLOW = 5,
     EVM_STACK_UNDERFLOW = 6,
     EVM_REVERT = 7,                ///< Execution terminated with REVERT opcode.
+    EVM_INVALID_IN_STATIC_MODE = 8,
```
Comment (chfast) [inline include/evm.h]: How about `EVM_STATIC_MODE_INVALIDATED` or `EVM_STATIC_MODE_ERROR`?

Comment (axic) [reply include/evm.h]: The second one sounds better.

**Comment by axic** [comment]
@chfast do you want to add support for this in the JIT or leave it to a separate PR?

**Comment by chfast** [comment]
No, EVMJIT does not support any of these.

**Comment by axic** [comment]
@chfast it easily could, `Compiler.cpp` can have another goto for that :)

**Pull request merged**